Song_sort_Genre.php

```diff
@@ -42,7 +73,10 @@
 			<h3>Genre</h3>
 			<h3>Duration</h3>
 			<h3>Size</h3>				
+		</div>
+		<div class="content_4">
 			<?php
+				//Called up all data to display in the page and information of each column is place underneath their title
 				require "musicdatabase_database_mysqli.php";
 					
 				$query = ("SELECT s.`Title`, GROUP_CONCAT(DISTINCT g.`Genre` SEPARATOR ', ') AS 'Genre',
```

Comment:
21/9: This the query that use to sort all the data and display it as a structure data(with older by genre and artist in the order from A to Z)